### PR TITLE
gsoc-ml+gitlab-merge+releases-patch-1

### PR DESCRIPTION
**Pull request by sgoggins**
This pull request fixes installation issues for the gitlab worker and releases worker.

### CODE REVIEW

File: augur/config.py
Comment (ccarterlandis) [inline augur/config.py:37]: What does this `all_focused` key actually do? I don't see it used anywhere in the code for these workers. Also, this is a separate issue, but I don't like the name `all_focused`. It's not descriptive at all and as a new user I would have no idea what it would change. The name doesn't need to convey literally everything it does, but something like `collect_all_issues` or `complete_collection` would be far more useful IMHO.

File: augur/config.py
Comment (ccarterlandis) [inline augur/config.py:42]: Is this **GitLab** merge requests, or just merge requests of any kind? Context clues tell me it's just for GitLab, so I think we need to be very specific both in the model and in the given. Merge request is of course a very common FOSS term and could create confusion if we do no use it carefully.

File: augur/config.py
Comment (ccarterlandis) [inline augur/config.py:61]: `all_focused` and merge request terminology comments apply here too.

File: augur/config.py
Comment (ccarterlandis) [inline augur/config.py:174]: This indentation is incorrect. The brackets appear too far too the left, and the code appears to be too far to the right.

File: augur/config.py
Comment (ccarterlandis) [inline augur/config.py:276]: This indentation is also incorrect. It needs to follow the formatting of the rest of the object.

File: schema/generate/33-schema_update_35.sql
Comment (ccarterlandis) [inline schema/generate/33-schema_update_35.sql:4]: This needs to be added to the oneshot schema install script. And probably some other stuff we forgot to add too...

File: scripts/install/nltk_dictionaries.sh
Comment (ccarterlandis) [inline scripts/install/nltk_dictionaries.sh:25]: This code is also present in PR #1049. I don't know which version is correct.

File: setup.py
Comment (ccarterlandis) [inline setup.py:53]: Please fix the indentation on this line.

**Review by ccarterlandis — DISMISSED** [review]

File: augur/config.py
Comment (sgoggins) [reply augur/config.py:37]: @gabe-heim picked the name. Its for the PR and issues workers to go make sure they are getting everything done.

File: augur/config.py
Comment (sgoggins) [reply augur/config.py:42]: This is *GitLab*

File: schema/generate/33-schema_update_35.sql
Comment (sgoggins) [reply schema/generate/33-schema_update_35.sql:4]: ok but that wouldn't prevent this from being merged. Where is the oneshot?

File: scripts/install/nltk_dictionaries.sh
Comment (sgoggins) [reply scripts/install/nltk_dictionaries.sh:25]: They are the same.  I've merged that commit in that other branch as well.

File: augur/config.py
Comment (ccarterlandis) [reply augur/config.py:42]: Okay, then I think we should change the model to `gitlab_merge_requests` and the given to `gitlab_url` to be as clear as possible.

**Review by sgoggins — COMMENTED** [review]
@ccarterlandis : Fixed all the indentation.

File: augur/config.py
Comment (ccarterlandis) [reply augur/config.py:37]: So what happens when it's off? Does it just...not collect everything? Like just a subset of the data or? 
And does this worker need it? I assume so, since it's about PRs (well MRs) but I guess what I'm trying to get at is that because the name of this key is confusing, I can't tell if this is actually supposed to be here or not.

File: augur/config.py
Comment (ccarterlandis) [reply augur/config.py:42]: I don't think is resolved, I still think this needs to be changed because it is confusing otherwise. Or at least, I haven't been convinced otherwise yet. 😉

File: augur/config.py
Comment (ccarterlandis) [reply augur/config.py:61]: This is also unresolved, for the same reasons as above.

File: schema/generate/33-schema_update_35.sql
Comment (ccarterlandis) [reply schema/generate/33-schema_update_35.sql:4]: Again, this is not resolved. It needs to be in the oneshot as we made that with the intention to keep it up to date with the small update scripts, otherwise we ship 2 different versions of the schema and that's not good for anyone. It's in the `schema/` folder, same as all the other schema files. It's called `20201025-Release-0.14.0.sql`

File: setup.py
Comment (ccarterlandis) [inline setup.py:53]: Please fix the formatting on this line.

File: scripts/install/nltk_dictionaries.sh
Comment (ccarterlandis) [reply scripts/install/nltk_dictionaries.sh:25]: This is not resolved either. Please fix the formatting as outlined in [this comment](https://github.com/chaoss/augur/pull/1049/files?file-filters%5B%5D=.sh#r544597459) on PR #1049.

**Review by ccarterlandis — CHANGES_REQUESTED** [review]
Request changes are in each comment thread.

File: augur/config.py
Comment (sgoggins) [reply augur/config.py:37]: You're asking questions about the basic design @gabe-heim has in place. I'm not doing anything new or special here. This is a switch to *ensure* that we get everything on the first run. Its been there for over two years.

File: augur/config.py
Comment (sgoggins) [reply augur/config.py:61]: Same answer as above. Its been there over two years. Now is not the time to stop a pull request because of it. Ask @gabe-heim .

File: scripts/install/nltk_dictionaries.sh
Comment (sgoggins) [reply scripts/install/nltk_dictionaries.sh]: extra space corrected.

File: setup.py
Comment (sgoggins) [reply setup.py]: fixed.

**Review by ccarterlandis — CHANGES_REQUESTED** [review]
Not everything is fixed and I still have several unanswered questions. By merging `gsoc-patch` you also introduced more code that has issues, so I have added my review to this new code as well. Not much, but it still needs to be fixed.
I will approve and merge it when all my questions, comments, and requested changes have been appropriately addressed.

File: augur/config.py
Comment (ccarterlandis) [reply augur/config.py:42]: After further thought this doesn't need to block this PR, as we should do the same thing for GitHub. We can do that later. But I still think it needs to be changed.

File: augur/config.py
Comment (sgoggins) [reply augur/config.py:37]: @gabe-heim : Please pull this branch and fix it, or merge your PR branch and I'll merge that here first.

**Pull request closed without merging**
